Sources/Orbit/Support/Environment Keys/TextIsCopyableKey.swift

```diff
@@ -0,0 +1,26 @@
+import SwiftUI
+
+struct TextIsCopyableKey: EnvironmentKey {
+    static let defaultValue = false
+}
+
+public extension EnvironmentValues {
+
+    /// A text copyable status stored in a view’s environment, used for Orbit text components, such as `Text` or `Heading`.
+    var textIsCopyable: Bool {
```

Comment:
I liked "selectable" better... is "copy" really the only option in all circumstances? Can't the text for example also be read by Siri or something?

Reply:
Me too, but SwiftUI already has a `textSelection` modifier that "Use this method to indicate if text is selectable". But this is in regards to Mac selectability of any subtext. 

We are allowing text selection here, we only allow user to "long tap" and "copy" the text.